README.md

```diff
@@ -188,6 +192,16 @@ o-tracking will
 * Automatically pickup ftsession from cookies for you.
 * Page events automatically pick up the url and the referrer.
 * Click events [initalised as above](#usage), will populate a `domPathTokens` property. If the clicked element has the `data-trackable` attribute, sibling elements will also be included within `domPathTokens`.
+* Component view events will be fired with elements set `data-o-tracking-view` attribute or passed as elementsToTrack[array] in `opts`. They will populate a `domPathTokens` as well as Click events. You could pass custom category and also callback to collect data for events.  
+	```js
+	const opts = {
+		category: 'custom', // default: 'component'
+		elementsToTrack: [ element, element, ... ], // default: elements set data-o-tracking-view
+		callback: () => {} // default: null
```

Comment:
Wondering if we could make the name of this more explicit, something like ... `getContextForComponent`  🤔 ?